transportable-udfs-plugin/src/main/java/com/linkedin/transport/plugin/Defaults.java

```diff
@@ -101,7 +107,13 @@ private static Properties loadDefaultVersions() {
 
   private static DependencyConfiguration getDependencyConfiguration(ConfigurationType configurationType,
       String module, String platform) {
-    return new DependencyConfiguration(configurationType,
-        module + ":" + DEFAULT_VERSIONS.getProperty(platform + "-version"));
+    return getDependencyConfiguration(configurationType, module, platform, null);
```

Comment:
Overload the method instead of passing null and checking inside with an if condition

Reply:
That would lead to code duplication. I am not sure if code duplication is better than having a simple if condition.